source/_components/sensor.folder.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "folder sensor"
+description: "Component for monitoring the contents of a folder."
+date: 2018-02-06 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: file.png
+ha_category: Sensor
+ha_iot_class: "Local Polling"
+ha_release: 0.63
+---
+
+Home-assistant component for monitoring the contents of a folder.
```

Comment:
`Home-assistant` -> `Home Assistant`.

In general, I'd suggest rewriting this sentence since it is kinda obvious that this component is for Home Assistant already 😉